### DISCUSSION TITLE
mistype in rules: BT-1541-Contract

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#689] [Q&A] [answered]
Hi Lab

I think there are mistypes in these rule messages (SDK 1.7.1):

```
<entry key="rule|text|BR-BT-01451-0100">'Date on which the winner was chosen' 
(BT-1541-Contract) must be earlier than 'Date of the conclusion of the contract' (BT-145-Contract)</entry>

<entry key="rule|text|BR-BT-01451-0101">'Date on which the winner was chosen' 
(BT-1541-Contract) must be earlier than 'Notice dispatch date' (BT-05(a)-notice)</entry>
```

"BT-1541-Contract" instead of "BT-1451-Contract" (Winner Decision Date)

Regards
Anna

**Comment by bertrand-lorentz** [comment, marked as answer]
Thanks for your feedback.
There is indeed a typo in those 2 messages. We will correct this in an upcoming version of the SDK.

**Reply by elerik** [reply]
Hi devs, 
just kind reminder ;)

**Comment by elerik** [comment]
rule|text|BR-OPT-00321-0101
it looks like it has also wrong typo 
<entry key="rule|text|BR-OPT-00321-0101">Any exisiting 'ID – Tender' (OPT-316-Tender) must be associated with at least one tender identifier reference (OPT-320-LotResult)</entry>

it should be 
<entry key="rule|text|BR-OPT-00321-0101">Any exisiting 'ID – Tender' (OPT-321-Tender) must be associated with at least one tender identifier reference (OPT-320-LotResult)</entry>

**Reply by YvesJo** [reply]
Thanks elerik. 
This is indeed a typo and will get fixed.
